hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -240,19 +244,27 @@ commitTx party utxo (initialInput, vkh) =
         & Api.addVkInputs [commit | Just (commit, _) <- [utxo]]
         & Api.addOutputs [commitOutput]
  where
-  initialWitness = Api.BuildTxWith $ Api.mkScriptWitness initialScript initialDatum initialRedeemer
-  initialScript = Api.fromPlutusScript' MockInitial.validatorScript
-  initialDatum = Api.mkDatumForTxIn $ MockInitial.datum vkh
-  initialRedeemer = Api.mkRedeemerForTxIn $ MockInitial.redeemer ()
-
-  commitOutput = Api.TxOut commitAddress commitValue commitDatum
-  commitScript = Api.fromPlutusScript MockCommit.validatorScript
-  commitAddress = Api.mkScriptAddress networkId commitScript
+  initialWitness =
+    Api.BuildTxWith $ Api.mkScriptWitness initialScript initialDatum initialRedeemer
+  initialScript =
+    Api.fromPlutusScript Api.AsPlutusScriptV1 MockInitial.validatorScript
+  initialDatum =
+    Api.mkDatumForTxIn $ MockInitial.datum vkh
+  initialRedeemer =
+    Api.mkRedeemerForTxIn $ MockInitial.redeemer ()
+
+  commitOutput =
+    Api.TxOut commitAddress commitValue commitDatum
+  commitScript =
+    Api.fromPlutusScript Api.AsPlutusScriptV1 MockCommit.validatorScript
+  commitAddress =
+    Api.mkScriptAddress networkId commitScript
```

Comment:
You like this better, or was one line too long and you made all of them two-liners to stay consistent?

Reply:
Both, but it's the latter which triggered the change as I had to cut some long lines.